rmarkdown/The Katak Detector.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Frog Discovery Tool: A monitoring tool to detect frog occurrences based on identified
+  sample frog species'
+author: "The Katak Detector"
+date: Nurhanie Syakirah (S2132021), Yu Yuen Hern (S2121801), Nor Shaffilza (S2156225), Aina Shamsuddin (S2138897), Dong Xiaoran (S2131591)
+output:
+  slidy_presentation: default
+  powerpoint_presentation: default
+  ioslides_presentation: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, cache=TRUE, warning=FALSE, message=FALSE)
+```
+
+## Background of the Project
+
+Amphibians such as frogs and toads are strong indicator species for pollution. They have permeable skin through which they absorb oxygen and toxins. As a result, they’re extremely sensitive to changes in the quality of air and water. They’re often the first animals, for example, to be affected by pesticide use in or near their ecosystems. Scientists look to them to get a sense of the overall health of old-growth forest ecosystems and to monitor the effects of human-caused habitat changes.
+
+Dataset: Frog occurrence dataset
+
+>https://challenge.ey.com/challenges/level-2-global-frog-discovery-tool-EEmnW-5fR/data-description
+
+
+## Introduction of Project
+
+- Problem statements
+
+The declination of frogs may be a warning to us that our planet is becoming unliveable. The disappearance of frogs is causing concerns such as increase of greenhouse effect, climate change & balance of ecosystem.
+
+- Research questions

```

Comment:
We didn't really answered these questions. Perhaps consider dropping those unanswered?